Dockerfile.manipulation.simulation

```diff
@@ -0,0 +1,45 @@
+ARG ROS_DISTRO=humble
+
+FROM husarion/rosbot-xl-gazebo:$ROS_DISTRO
+
+ENV IGNITION_VERSION fortress
+ENV HUSARION_ROS_BUILD simulation
+
+WORKDIR /ros2_ws
+
+# install everything needed
+RUN apt-get update --fix-missing && apt-get install -y \
+        python3-pip \
+        python3-colcon-common-extensions \
+        python3-rosdep \
+        python3-vcstool \
+        git && \
+    apt-get upgrade -y && \
+    rm -rf /ros2_ws/* && \
+    # build & install ROSbot XL packages
+    source "/opt/ros/$ROS_DISTRO/setup.bash" && \
+    git clone https://github.com/husarion/rosbot_xl_manipulation_ros.git /ros2_ws/src/rosbot_xl_manipulation_ros && \
+    vcs import src < src/rosbot_xl_manipulation_ros/rosbot_xl_manipulation/rosbot_xl_manipulation.repos && \
+    vcs import src < src/rosbot_xl_ros/rosbot_xl/rosbot_xl_hardware.repos && \
+    vcs import src < src/rosbot_xl_ros/rosbot_xl/rosbot_xl_simulation.repos && \
+    vcs import src < src/open_manipulator_x/open_manipulator_x.repos && \
+    rm -rf /etc/ros/rosdep/sources.list.d/20-default.list && \
+    rosdep init && \
+    rosdep update --rosdistro $ROS_DISTRO && \
+    rosdep install -i --from-path src --rosdistro $ROS_DISTRO -y && \
+    colcon build && \
+    # make the image smaller
+    export SUDO_FORCE_REMOVE=yes && \
+    apt-get remove -y \
+        python3-pip \
+        python3-colcon-common-extensions \
+        python3-rosdep \
+        python3-vcstool \
+        git && \
+    apt-get autoremove -y && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
```

Comment:
Is it okay to set `RMW` in Dockerfile? I can't see it in [Dockerfile.simulation](https://github.com/husarion/rosbot-xl-docker/blob/main/Dockerfile.simulation).

Reply:
I think that it won't be a problem - it will just set a default value (which otherwise would be fastdds), and we overwrite it in compose files

Reply:
Out of curiosity I checked, and we used to also set it in Dockerfile.simulation, but it was removed.